Hoverchairs

ItsMeThom opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
Added hoverchairs. A hovering wheelchair with lowered friction to let people levitate around the station. Its *slightly* flaster than a wheelchair, but the lower friction means it can pick up speed, though its much slower than other vehicles.

Its a Tier 2 Experimental Research (7500 Research) called "Hover Assistance Technology" and it's buildable at the exosuit fabricator in robotics for:
- 5 Steel
- 3 Gold
- 2.5 Plasma

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Its the future, we need chairs that hover dammit!

## Technical details
<!-- Summary of code changes for easier review. -->
No code changes, all YAML; Added the object definition, the research, lathe recipe.
## Media
Research and lathe:
![exosuit_recipe](https://github.com/user-attachments/assets/3f271548-6629-4cf0-ae7a-31b4dd6c7405)
![research screen](https://github.com/user-attachments/assets/cbb400fb-9781-44a8-862c-74ad7b9aafdb)

Sprite:
![hover_front](https://github.com/user-attachments/assets/39ed847e-7b16-481b-9119-14b5d5489ebb)

![hover side](https://github.com/user-attachments/assets/988ed8e2-3f66-430f-813d-556191e63e33)

![hover back](https://github.com/user-attachments/assets/f95c3251-c769-4a09-9ea3-877192d19760)


Video of it in action:

https://github.com/user-attachments/assets/a85f3fb7-337b-40c5-a3a0-bbd33e18e3ba




## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
:cl:
- add: Added hoverchairs!

DEB-DEB0002 commented:
new syndicate item. Evil Hoverchair for the evil villian... Does it works in space?

ItsMeThom commented on Resources/Prototypes/_Goobstation/Entities/Objects/Vehicles/vehicles.yml:
```diff
@@ -240,3 +240,35 @@
       folded: true
+
+
+- type: entity
+  id: VehicleHoverchair
+  parent: [ BaseVehicle, BaseItem ]
+  name: hoverchair
+  description: A luxury hoverchair, kick back and relax.
+  components:
+  - type: Sprite
+    sprite: _GoobStation/Objects/Vehicles/hoverchair.rsi
+    layers:
+    - state: vehicle
+    noRot: true
+  - type: Vehicle
+    requiredHands: 0
+    engineRunning: true
+    hornSound:
+      collection: BikeHorn
+      params:
+        variation: 0.125
+    renderOver: North, NorthEast, NorthWest, East, West
+  - type: MovementSpeedModifier
+    acceleration: 10
+    friction: 0.5
+    baseSprintSpeed: 3.5
+    baseWalkSpeed: 2.5
+  - type: StaticPrice
+    price: 175
+  - type: PhysicalComposition
+    materialComposition:
+      Steel: 125
+      Plastic: 125
```
its deliberate, you silly ai

ItsMeThom commented:
> new syndicate item. Evil Hoverchair for the evil villian... Does it works in space?

No it doesnt provide any movement in space, it is a vehicle like the wheelchair so no off-grid propulsion.

Future PR though, for Evvviiill Hoverchair (In red!)

PunishedJoe commented:
lol that is NOT worth 7500 points

DEB-DEB0002 commented:
> No it doesnt provide any movement in space, it is a vehicle like the wheelchair so no off-grid propulsion.
> 
> Future PR though, for Evvviiill Hoverchair (In red!)

think about evil hovechair that allow to fly in space, its cool.

pheenty commented:
> lol that is NOT worth 7500 points

Agreed.

That needs to be moved to an existing technology, might even move it to prescription glasses (with renaming) as it shares a theme of helping people with disabilities. Also shis should ve printable on medfab and maybe on protolathe, this has nothing to do with robotics

SolsticeOfTheWinter commented on Resources/Prototypes/_Goobstation/Research/experimental.yml:
```diff
@@ -14,3 +14,15 @@
   - ModsuitShell
   - ModsuitPlatingExternal
+
+- type: technology
```
Now here's an idea, as suggested by previous people.
Move the prescription glasses and this to a t1/t2 civ tech, condensing it to a tech about accessibility or something.
Lord knows we don't need more bloat researches, the sci mains ALREADY hate me.

ItsMeThom replied:
Sure, I can do that when Im home from work today @SolsticeOfTheWinter.

I'll move this and prescription glasses into the same tech, something like "Crew Assistance" or "Space Accessibility". Will that be fine as a Tier 2, or dyou think these would better serve as a Tier 1? I'm a little hesitant to make this Tier 1 because this is essentially a vehicle.

ItsMeThom commented:
@SolsticeOfTheWinter I've combined prescription glasses/hoverchair research item into a 5000pt T1 as you suggested. Tested and working in game.

ItsMeThom commented:
> Also shis should be printable on medfab and maybe on protolathe, this has nothing to do with robotics

Id rather keep it robotics, its arguably in the same category as mechsuits/modsuits, its a crew operated vehicle, its robotics area.

ItsMeThom commented:
Ok, I've moved the hoverchair into a `MedicalVehiclesAdvanced` pack with the latest merge. I also removed it from the exosuit fab @pheenty  and Ive added it to the `dynamicPacks` for the med techfab and protolathe.

HenhenIII commented:
At this point, just add WALL·E also.
